cmd/controller/main.go

```diff
@@ -75,6 +76,11 @@ func main() {
 	clientSet := kubernetes.NewForConfigOrDie(manager.GetConfig())
 	cloudProvider := registry.NewCloudProvider(cloudprovider.Options{ClientSet: clientSet})
 
+
+	if err := expiration.NewController(manager.GetClient()).Register(manager); err != nil {
```

Comment:
I understand that we're going to be moving this out eventually. Would you be able to put these with the rest of controllers until you change it?

Reply:
Nope -- different interfaces.